### PR TITLE
Network: firewall controller rules

### PR DESCRIPTION
**Pull request by cheina97**
# Description

This PR introduces the support for rules in firewallconfiguration controller

### CODE REVIEW

File: pkg/firewall/firewallconfiguration_controller.go [hunk 1]
```diff
@@ -113,15 +115,13 @@ func (r *FirewallConfigurationReconciler) Reconcile(ctx context.Context, req ctr
...
 
-	// Enforce table existence and apply chains.
+	// Enforce table existence.
 	table := addTable(r.NftConnection, &fwcfg.Spec.Table)
 
-	for i := range fwcfg.Spec.Table.Chains {
-		if err = addChain(r.NftConnection, &fwcfg.Spec.Table.Chains[i], table); err != nil {
-			return ctrl.Result{}, err
-		}
+	if err := addChains(r.NftConnection, fwcfg.Spec.Table.Chains, table); err != nil {
```
Comment (aleoli) [inline pkg/firewall/firewallconfiguration_controller.go:123]:
```suggestion
	if err = addChains(r.NftConnection, fwcfg.Spec.Table.Chains, table); err != nil {
```

File: pkg/liqo-controller-manager/webhooks/firewallconfiguration/firewallconfiguration.go [hunk 2]
```diff
@@ -12,33 +12,52 @@
...
+// +kubebuilder:rbac:groups=networking.liqo.io,resources=firewallconfigurations,verbs=get;list;watch;update;patch
 
 type fwcfgwh struct {
 	decoder *admission.Decoder
 }
 
+type fwcfgwhm struct {
+	fwcfgwh
+}
+type fwcfgwhv struct {
```
Comment (aleoli) [inline pkg/liqo-controller-manager/webhooks/firewallconfiguration/firewallconfiguration.go:40]: Can you use some more meaningful names?

**Review by aleoli — APPROVED** [review]

File: pkg/liqo-controller-manager/webhooks/firewallconfiguration/errror.go
Comment (francescodanzi) [inline pkg/liqo-controller-manager/webhooks/firewallconfiguration/errror.go:1]: I think there is a typo in the filename

**Review by francescodanzi — APPROVED** [review]

**Pull request merged**